README.md

```diff
@@ -20,30 +20,6 @@ More detailed documentation and info on writing declarations can be found [on th
 
 `tstl -p path/to/tsconfig.json`
 
-**Options**
-```
-tstl [options] [files...]
-
-In addition to the options listed below you can also pass options for the
-typescript compiler (For a list of options use tsc -h).
-
-NOTES:
-- The tsc options might have no effect.
-- Options in tsconfig.json are prioritized.
```

Comment:
I think `Options in tsconfig.json are prioritized.` should be in here or the wiki.

Reply:
Changed it so that CLI is prioritized. This is the same behaviour now as in `tsc` and probably any cli tool out there. So I don't think its **needed** to mention that, but it wouldn't hurt either if this is mentioned somewhere in the docs.

Reply:
That's much better. It only needed mentioning because it was different from tsc.